mifospay/src/main/java/org/mifos/mobilewallet/mifospay/standinginstruction/ui/SIDetailsActivity.kt

```diff
@@ -58,6 +65,8 @@ class SIDetailsActivity : BaseActivity(), StandingInstructionContract.SIDetailsV
         setUpUI()
     }
 
+    @SuppressLint("SimpleDateFormat")
+    @RequiresApi(Build.VERSION_CODES.O)
```

Comment:
@haran2248 setting the min date should work on all the devices supported, hence requiring API version26+ for this is probably not the best way. Can you figure out any other way of doing this?

Reply:
yeah ill work on it

Reply:
The requires api part was unnecessary i guess

Reply:
I think it's needed. See there's a need to suppress the warning for using a feature not supported (or introduced later) by the specified min SDK version. Did you explore other methods?

Reply:
Yeah I just found a simple fix to that warning by adding locale.getDefault(),Now the warrning has disappeared,and it works fine without any additional support